base/download.jl

```diff
@@ -39,8 +39,9 @@ function download_curl(curl_exe::AbstractString, url::AbstractString, filename::
     err = PipeBuffer()
     process = run(pipeline(`$curl_exe -s -S -g -L -f -o $filename $url`, stderr=err), wait=false)
     if !success(process)
-        stderr = readline(err)
-        error(stderr)
+        error_msg = readline(err)
+        @error "Download Failed" error_msg
```

Comment:
this should've been a string interpolation, fixed in https://github.com/JuliaLang/julia/pull/31620